docs/user_guide_ova.md

```diff
@@ -0,0 +1,190 @@
+* Manage Harbor system if you are the system administrator:
+ + Manage users.
+ + Manage destinations.
+ + Manage replication policies.
+* Pull and push images using Docker client.
+* Delete repositories and images.
+
+
+##Role Based Access Control
+
+![rbac](img/rbac.png)
+
+Harbor manages images through projects. Users can be added into one project as a member with different privileges. RBAC (Role Based Access Control) is provided in Harbor and there are four roles with different privileges:  
+
+* **Guest**: Guest has read-only privilege for a specified project.
+* **Developer**: Developer has read and write privileges for a project.
+* **ProjectAdmin**: When creating a new project, you will be assigned the "ProjectAdmin" role to the project. Besides read-write privileges, the "ProjectAdmin" also has some management privileges, such as adding and removing members.
+* **SysAdmin**: "SysAdmin" has the most privileges. In addition to the privileges mentioned above, "SysAdmin" can also list all projects, set an ordinary user as administrator and delete users. The public project "library" is also owned by the administrator.  
+* **Anonymous**: When a user is not logged in, the user is considered as an "anonymous" user. An anonymous user has no access to private projects and has read-only access to public projects.  
+
+##User account
+Harbor supports two authentication mode:  
+
+* **Database**  
+
+	Users are stored in the local database.  
+	
+	You can determine whether the self-registration is allowed or not under this authentication mode by setting **"Self Registration"** according to the [installation guide](installation_guide_ova.md). If this flag is set to off users can only be added by the system administrators.  
+	
+	When register or add new users, the username and email must be unique in the Harbor system. The password must contain at least 8 characters with 1 lowercase letter, 1 uppercase letter and 1 numeric character.  
+	
+	When you forgot your password, you can follow the below steps to reset the password:  
+
+	1. Click the link "forgot password" in the sign in page.  
+	2. Input the email used when you signed up, an email will be sent out to you.  
+	3. After receiving the email, click on the link in the email which directs you to a password reset web page.  
+	4. Input your new password and click "Submit".  
+	
+* **LDAP/AD**  
+
+	Using this authentication mode, users whose credentials are stored in an LDAP or AD server can log in to Harbor directly.  
+	
+	When an LDAP/AD user tries to login, Harbor will bind the LDAP/AD server with the **"LDAP Search DN"** and **"LDAP Search Password"** described in [installation guide](installation_guide_ova.md) first, if successes, Harbor will search users whose **"LDAP UID"** is equal to the username under **"LDAP Base DN"**.  
+	
+	Self-registration, password-modification and password-resetting are not supported anymore under LDAP/AD authentication mode.  

```

Comment:
Self-registration, changing password and resetting password are not supported anymore under LDAP/AD authentication mode because the users are managed by LDAP or AD.